src/components/workspace/WorkspaceSidebar.svelte

```diff
@@ -0,0 +1,344 @@
+                        }}
+                      >
+                        <FilePlus size={16} /> New Sequence
+                      </div>
+                    </DropdownMenu.Item>
+                    <DropdownMenu.Item on:click={onNewFolder}>
+                      <div
+                        class="flex cursor-pointer gap-1"
+                        use:permissionHandler={{
+                          hasPermission: hasEditWorkspacePermission,
+                          permissionError: 'You do not have permission to edit this workspace',
+                        }}
+                      >
+                        <FolderPlus size={16} /> New Folder
+                      </div>
+                    </DropdownMenu.Item>
+                    <DropdownMenu.Separator />
+                    <DropdownMenu.Item on:click={onImportFile}>
+                      <div
+                        class="flex cursor-pointer gap-1"
+                        use:permissionHandler={{
+                          hasPermission: hasEditWorkspacePermission,
+                          permissionError: 'You do not have permission to edit this workspace',
+                        }}
+                      >
+                        <ArrowUpFromLine size={16} />Import File
+                      </div>
+                    </DropdownMenu.Item>
+                  </DropdownMenu.Content>
+                </DropdownMenu.Root>
+              </div>
+            </div>
+          </Sidebar.Header>
+          <Sidebar.Content>
+            <Sidebar.Group className="p-0 h-full">
+              <Sidebar.GroupContent className="h-full">
+                <Sidebar.Menu className="h-full">
+                  {#if workspaceTree}
+                    <WorkspaceTreeView
+                      selectedTreeNodePath={selectedFilePath}
+                      treeNode={workspaceTree}
+                      {workspace}
+                      {user}
+                      on:nodeClicked
+                      on:nodeDelete
+                      on:nodeMove
+                      on:nodeRename
+                      on:newFolder
+                      on:newSequence
+                      on:importFile
+                      on:copyFileLocation
+                      on:moveToWorkspace
+                    />
+                  {:else}
+                    <div class="p-2 text-sm text-muted-foreground">No workspace loaded</div>
+                  {/if}
+                </Sidebar.Menu>
+              </Sidebar.GroupContent>
+            </Sidebar.Group>
+          </Sidebar.Content>
+        </div>
+      </Tabs.Content>
+      <Tabs.Content value="grid" class="mt-0 h-full" style="min-height: 300px;">
+        <div class="grid h-full grid-rows-[min-content_auto]">
+          <Sidebar.Header className="p-0">
+            <div class="flex items-center justify-between gap-0 border-b border-border bg-background p-[8px]">
+              <SectionTitle>Workspace Table View</SectionTitle>
```

Comment:
What is the reasoning for having distinct tabs for the table and the tree? Could this just be a single tab with a tree/grid switcher? Alternatively could there just be a tree with the ability to sort by name/date/etc as well as a filter on the tree?